pkgs/halo2-binary-merkle-tree-worker/src/worker.ts

```diff
@@ -15,8 +15,8 @@ export const halo2BinaryMerkleTreeWorker: IHalo2BinaryMerkleTree = {
     const response = await fetch(wasmModuleUrl)
     const bufferSource = await response.arrayBuffer()
 
-    await halo2BinaryMerkleTreeWasm.initSync(bufferSource)
-    await halo2BinaryMerkleTreeWasm.initPanicHook()
+    halo2BinaryMerkleTreeWasm.initSync(bufferSource)
+    halo2BinaryMerkleTreeWasm.initPanicHook()
```

Comment:
`initSync` (as the name suggests) and `initPanicHook` don't return Promises. So no need to `await` them.
awaiting them should not cause any issue though (so no bug)

Reply:
> `initSync` (as the name suggests) and `initPanicHook` don't return Promises. So no need to `await` them. awaiting them should not cause any issue though (so no bug)

That is theoretically true, but we need to test this first e2e on the UI.

Reply:
I tested the changes and all works as expected.